### DISCUSSION TITLE
Show Dropdown Options When Dropdown is Added

### DISCUSSION THREAD

**Discussion by rejogc** [flet-dev/flet#2681] [Q&A]
### Question

I'd like to use a Dropdown control to show the user Options from which to select. Is it possible to add a Dropdown with all it's options showing? This would save the user having to figure they have to press the Dropdown Icon Suffix to see all the Options and select one.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
In this SO thread are suggestions (hacks?) to achieve this in Flutter: https://stackoverflow.com/questions/57529394/how-to-open-dropdownbutton-when-other-widget-is-tapped-in-flutter
I'm afraid there is no official way to do this atm. 😕
